### PR TITLE
Feature: Notifications service

### PR DESCRIPTION
**Pull request by galmail**
This PR includes all the endpoints for the notifications service
NOTE: I noticed that the error codes are missing

### CODE REVIEW

File: src/services/notifications/notifications.ts [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+      .data;
+  },
+
+  /**
+   * Find By Id
+   * @param id the Id to search for
+   * @param rql an optional rql string
+   * @returns the first element found
+   */
+  async findAllById(
```
Comment (jensdev) [inline src/services/notifications/notifications.ts:88]: Might sound confusing? Maybe we can discuss naming during call? In other services there's just one 1 endpoint for your own / others

File: src/services/notifications/index.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+export type NotificationsService = ReturnType<typeof notifications> &
+  ReturnType<typeof health> & {
+    settings: ReturnType<typeof settings>;
+  };
+
+export const notificationsService = (
+  httpWithAuth: HttpInstance
+): NotificationsService => {
+  const client = httpClient({
+    basePath: NOTIFICATIONS_BASE,
```
Comment (jensdev) [inline src/services/notifications/index.ts:17]: Add `    transformRequestData: decamelizeKeys`

File: src/services/notifications/notifications.ts (hunk 1 above)
Comment (galmail) [reply src/services/notifications/notifications.ts]: cool, has been removed now following the discussion with Kobe

File: src/services/notifications/index.ts (hunk 2 above)
Comment (galmail) [reply src/services/notifications/index.ts:18]: ah right, done!

**Review by jensdev — APPROVED** [review]

**Pull request merged**